### PR TITLE
Set VPC on instance creation using VPC name

### PR DESCRIPTION
**Pull request by hnord-vdx**
Adding the `VPCUUID` field allows you to set a VPC of the droplet in the config.json. This way you can set VPC at instance creation without having to do a snapshot and then adding it to non-default VPC

### CODE REVIEW

**Comment by hnord-vdx** [comment]
@eyberg I see you are importing the `google/uuid` package perhaps we could add a check (`uuid.Parse(vpcName)`) if the name is a UUID and then skip the whole `GetVPC` round trip?  

Also, currently my implementation matches the LAST vpc based on the name. Maybe we want to match first VPC and warn using `Debugf` if there are other matches?

**Comment by eyberg** [comment]
i'd assume the vpc name is unique and so you really only need the first match? also i'm not crazy about passing in the id optionally but it appears we already do that in aws https://github.com/nanovms/ops/blob/master/provider/aws/aws_network.go#L163 so i'm ok w/allowing that if that's easier

**Comment by hnord-vdx** [comment]
Yeah, you are right. I just double checked vpc names have to be unique within a account so the zone check is not needed.

File: provider/digitalocean/digital_ocean_instance.go [hunk 1]
```diff
@@ -61,10 +62,23 @@ func (do *DigitalOcean) CreateInstance(ctx *lepton.Context) error {
 		}
 	}
 
+	// if users set VPC uuid instead of name it avoids lookup
+	vpcUUID := config.CloudConfig.VPC
+	if _, err := uuid.Parse(vpcUUID); err != nil {
```
Comment (eyberg) [inline provider/digitalocean/digital_ocean_instance.go:67]: can skip al of this if user didn't set it

File: provider/digitalocean/digital_ocean_network.go [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+func (do *DigitalOcean) GetVPC(ctx *lepton.Context, vpcName string) (*godo.VPC, error) {
+
+	if vpcName == "" {
+		return nil, nil
+	}
+
+	page := 1
+	var vpc *godo.VPC
+
+	// ctx.Logger().Debug("getting all vpcs")
```
Comment (eyberg) [inline provider/digitalocean/digital_ocean_network.go:23]: yank

**Comment by eyberg** [comment]
thanks!

**Pull request merged**